Classes/KIFUIViewTestActor.m

```diff
@@ -0,0 +1,396 @@
+//
+//  KIFUIViewActor.m
+//  KIF
+//
+//  Created by Alex Odawa on 1/21/15.
+//
+//
+
+#import "KIFUIViewTestActor.h"
+#import "UIWindow-KIFAdditions.h"
+
+
+@interface KIFUIViewTestActor ()
+
+@property (nonatomic, strong, readonly) KIFUITestActor *actor;
+@property (nonatomic, strong, readwrite) NSPredicate *predicate;
+
+@end
+
+
+@implementation KIFUIViewTestActor
+
+#pragma mark - Initialization
+
+- (instancetype)usingPredicate:(NSPredicate *)predicate;
+{
+
+    [self _appendPredicate:predicate];
+    return  self;
+}
+
+- (instancetype)usingAccessibilityLabel:(NSString *)accessibilityLabel;
+{
+    int systemVersion = [UIDevice currentDevice].systemVersion.intValue;
+
+    if ([accessibilityLabel rangeOfString:@"\n"].location == NSNotFound || systemVersion == 6) {
+        return [self usingPredicate:[NSPredicate predicateWithFormat:@"accessibilityLabel MATCHES %@", accessibilityLabel]];
```

Comment:
Using MATCHES implies the right hand expression is a regex. Why not use '==' instead? The right hand expression would always be a accessibilityLabel, accessibilityIdentifier, traits or value.